lib/functions.inc.php

```diff
@@ -363,6 +363,31 @@ function change_password( $ldap, $dn, $password, $ad_mode, $ad_options, $samba_m
     return $result;
 }
 
+
+# Change sshPublicKey attribute
+# @return result code
+function change_sshkey( $ldap, $dn, $sshkey ) {
```

Comment:
We need to pass $change_sshkey_attribute as parameter of function change_sshkey, else it will be unef